src/terrasoApi/shared/memberships/membershipsFragments.ts

```diff
@@ -83,6 +25,10 @@ export const groupMembersPending = /* GraphQL */ `
 
 export const groupMembersInfo = /* GraphQL */ `
   fragment groupMembersInfo on GroupNode {
+    id
```

Comment:
Question: Do we need to add these group fields here? I always see this fragment used next to the groupFields fragment, or where is this needed?

Reply:
it would be needed for non-group code, i.e. when conceptually thinking of `GroupNode` as `MembershipList`. i should update the fragment names to reflect this. the example right now is `defaultGroupWithMembersSample` in `landscapeFragments` needs to manually remember to add `id` and `slug`, and same with project queries eventually